localisation/00_zuwc_united_localisation.csv

```diff
@@ -3041,16 +3041,22 @@ nick_the_unstoppable;the Unstoppable;;;;;;;;;;;;;x
 nick_the_lightning_fist;the Lightning-Fist;;;;;;;;;;;;;x
 DEATH_WANDERING_ISLE_NO_KILLER;went to the Wandering Isle;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+mists_of_pandaria_dont_block_interactions_tooltip;Mists of Pandaria do NOT block interactions;;;;;;;;;;;;;x
+EVTTITLE_WCPAN_100;The Mists of Pandaria;;;;;;;;;;;;;x
+EVTDESC_WCPAN_100_azeroth;This morning the messenger brought you astounding news. A group of boats fishing in the South Seas has stumbled upon a land where it shouldn't be. One of the ships wrecked, but the rest managed to find a way out of mists.\n\nAll ships were aware of avoiding those waters due to dense fog and sea legends, but it looks like the mists have parted.\n\nThe fishermen even tell that they have seen gold towers far on the horizon. If their stories aren't fake, they could discover a new land inhabited by sentient beings right south of the Maelstrom. We should send expeditions to make sure.;;;;;;;;;;;;;x
+EVTOPTA_WCTRO_100_conquer;New lands to explore and conquer.;;;;;;;;;;;;;x
+EVTOPTA_WCTRO_100_welcome;Welcome, wanderers!;;;;;;;;;;;;;x
+EVTOPTA_WCPAN_100_expeditions;Send expeditions, I want to know more.;;;;;;;;;;;;;x
+EVTDESC_WCPAN_100_pandaria;This morning the messenger brought you astounding news. A group of lighthouse keepers has noticed strange boats coming out of the fog. One of the ships wrecked, but the rest managed to find a way out of the mists.\n\nHow could these boats find a way to Pandaria and out? May it mean that the mists have parted?\n\nThe wrecked ship's crew is not like any other race we know. They speak about kingdoms and places beyond the sea. If their stories aren't fake, the world is much bigger and more dangerous than we thought. We must be prepared to face more ships and wanderers soon.;;;;;;;;;;;;;x
+EVTDESC_WCCLN_113;Unfortunately, the marriage between [target_other_spouse.GetDynName] and [target_spouse.GetDynName] is impossible. [Root.GetDivorceReason];;;;;;;;;;;;;x
+GetDivorceReason_mists;You sent messengers to [target_other_spouse.GetFirstName], but they disappeared in the mists of the South Seas.;;;;;;;;;;;;;x
+GetDivorceReason_cant_marry;Due to [target_spouse.GetHerHis] race, [target_spouse.GetFirstName] can't marry and love someone [target_spouse.GetSheHe] wish.;;;;;;;;;;;;;x
+GetDivorceReason_wrong_race;Due to the different physiology and incompatibility of their races, they can't marry and love each other.;;;;;;;;;;;;;x
+RULE_MISTS_OF_PANDARIA;�RWarCraft�!: Mists of Pandaria;;;;;;;;;;;;;x
+RULE_MISTS_OF_PANDARIA_ON;On;;;;;;;;;;;;;x
+RULE_MISTS_OF_PANDARIA_ON_DESC;The Mists of Pandaria are enabled, making Pandaria isolated till they're gone around the 613th year.;;;;;;;;;;;;;x
+RULE_MISTS_OF_PANDARIA_OFF;Off;;;;;;;;;;;;;x
+RULE_MISTS_OF_PANDARIA_OFF_DESC;The Mists of Pandaria are disabled thus Pandaria isn't isolated.;;;;;;;;;;;;;x
```

Comment:
```suggestion
RULE_MISTS_OF_PANDARIA_OFF_DESC;The Mists of Pandaria are disabled, making Pandaria open to the world.;;;;;;;;;;;;;x
```

Reply:
Done in 33a5b5203922313bb6a9824fef205b46499ecbcb.